features/projects/components/project-card/project-card.tsx

```diff
@@ -17,9 +16,9 @@ const languageNames = {
 };
 
 const statusColors = {
-  [ProjectStatus.stable]: BadgeColor.success,
-  [ProjectStatus.warning]: BadgeColor.warning,
-  [ProjectStatus.critical]: BadgeColor.error,
+  [ProjectStatus.error]: { color: BadgeColor.error, label: "Critical" },
+  [ProjectStatus.warning]: { color: BadgeColor.warning, label: "Warning" },
+  [ProjectStatus.info]: { color: BadgeColor.success, label: "Stable" },
```

Comment:
That's exactly what I wanted to see here 😄 No workarounds or so. Clean and straightforward solution.